test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -73,20 +72,20 @@ public void testPointSplitting() {
     assertArrayEquals(yCoords, spyRouteDrawer.getValues(dummyPoints, point -> point.getY()), 0);
   }
 
-  @Test
-  public void testPointAcquisition() {
-    assertArrayEquals(dummyPoints,
-        spyRouteDrawer.getRoutePoints(dummyRouteDescription, dummyMapData, 0, 0, new Dimension(), new Dimension()));
-    for (int i = 0; i < 10; i++) {
-      assertArrayEquals(dummyPoints, spyRouteDrawer.getRoutePoints(dummyRouteDescription, dummyMapData, randomInt(),
-          randomInt(), new Dimension(), new Dimension()));
-      assertArrayEquals(dummyPoints, spyRouteDrawer.getRoutePoints(dummyRouteDescription, dummyMapData, 0, 0,
-          new Dimension(randomInt(1000, 100), randomInt(1000, 100)), new Dimension()));
-      final int randX = randomInt(1000, 0);
-      assertEquals(new Point(randX, 0),
-          MapRouteDrawer.getPointOnMap(new Point(randX, 0), randX, 0, new Dimension(randX, 0), new Dimension()));
-    }
-  }
+//  @Test
```

Comment:
This is no longer working since the method does no longer exist... Will remove on test fixing